src/app/personside/infotabs/varsel/Varsler.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { useState } from 'react';
+import { Varsel, Varseltype } from '../../../../models/varsel';
+import { datoSynkende, formatterDato } from '../../../../utils/dateUtils';
+import styled from 'styled-components';
+import theme from '../../../../styles/personOversiktTheme';
+import { createTable } from '../../../../utils/tableUtils';
+import Ekspanderbartpanel from 'nav-frontend-ekspanderbartpanel';
+import MeldingsListe from './meldingsliste/MeldingsListe';
+import VisuallyHiddenAutoFokusHeader from '../../../../components/VisuallyHiddenAutoFokusHeader';
+import { Element, Normaltekst } from 'nav-frontend-typografi';
+import { Bold } from '../../../../components/common-styled-components';
+
+interface VisningProps {
+    varsler: Varsel[];
+}
+
+const TableStyle = styled.div`
+    table {
+        width: 100%;
+        text-align: right;
+        thead {
+            text-transform: uppercase;
+            th {
+                margin-bottom: 0.5rem;
+            }
+            th:nth-child(3) ~ th {
+                ${theme.visuallyHidden};
+            }
+        }
+        tbody tr {
+            ${theme.hvittPanel};
+            margin-bottom: 0.5rem;
+            display: block;
+            td {
+                &:last-child {
+                    width: 100%;
+                }
+            }
+        }
+        th,
+        td {
+            display: inline-block;
+            &:not(:last-child) {
+                vertical-align: bottom;
+                padding: 1rem 1rem 0;
+            }
+            &:first-child {
+                text-align: left;
+                width: 20%;
+            }
+            &:nth-child(2) {
+                width: 50%;
+            }
+            &:nth-child(3) {
+                width: 30%;
+            }
+        }
+    }
+`;
+
+function lagVarselTabellRow(varsel: Varsel) {
+    const [åpen, setÅpen] = useState(false);
+    const dato = formatterDato(varsel.mottattTidspunkt);
+    const varseltype = <Bold>{Varseltype[varsel.varselType]}</Bold>;
+    const distinkteKommunikasjonsKanaler = Array.from(new Set(varsel.meldingListe.map(melding => melding.kanal))).join(
```

Comment:
Denne onelineren ville jeg gjerne ha gitt litt newlines for å gjøre den mer lesbar

Reply:
Tror prettier har herjet litt her, men kan jo kanskje trekkes ut i en funksjon..?

Reply:
kanskje, jeg synes ikke det der var pretty ivertfall :D